openpype/settings/entities/schemas/projects_schema/schemas/template_nuke_knob_inputs.json

```diff
@@ -0,0 +1,258 @@
+        "type": "collapsible-wrap",
+        "label": "{label}",
+        "collapsible": true,
+        "collapsed": true,
+        "children": [{
+            "type": "list",
+            "key": "{key}",
+            "object_type": {
+                "type": "dict-conditional",
+                "enum_key": "type",
+                "enum_label": "Type",
+                "enum_children": [
+                    {
+                        "key": "text",
+                        "label": "Text",
+                        "children": [
+                            {
+                                "type": "text",
+                                "key": "name",
+                                "label": "Name"
+                            },
+                            {
+                                "type": "text",
+                                "key": "value",
+                                "label": "Value"
+                            }
+                        ]
+                    },
+                    {
+                        "key": "formatable",
+                        "label": "Formate from template",
+                        "children": [
+                            {
+                                "type": "text",
+                                "key": "name",
+                                "label": "Name"
+                            },
+                            {
+                                "type": "text",
+                                "key": "template",
+                                "label": "Template",
+                                "placeholder": "{{key}} or {{key}};{{key}}"
+                            },
+                            {
+                                "type": "enum",
+                                "key": "to_type",
+                                "label": "Knob type",
+                                "enum_items": [
+                                    {
+                                        "text": "Text"
+                                    },
+                                    {
+                                        "number": "Number"
+                                    },
+                                    {
+                                        "decimal_number": "Decimal number"
+                                    },
+                                    {
+                                        "2d_vector": "2D vector"
+                                    }
+                                ]
+                            }
+                        ]
+                    },
+                    {
+                        "key": "hex",
+                        "label": "Hexadecimal (0x)",
+                        "children": [
+                            {
+                                "type": "text",
+                                "key": "name",
+                                "label": "Name"
+                            },
+                            {
```

Comment:
Wouldn't be easier to use `"color"` instead of `"text"`?

Reply:
good point, thanks ;)